tests/test_macrocycle.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import rdkit.Chem.AllChem as rdkit
+import stk
+
+
+def test_cycle_atoms():
+    cycle = rdkit.MolFromSmiles('CCCC1CCCCCCCCC1')
+    cycle = rdkit.AddHs(cycle)
+    rdkit.EmbedMolecule(cycle)
+
+    cycle_su = stk.MacrocycleStructUnit(cycle, [])
+    assert set(cycle_su.cycle_atoms()) == set([3, 4, 5, 6, 7, 8,
+                                               9, 10, 11, 12])
```

Comment:
This is an inefficient way of creating a set, because you first create a list and then you convert the list into a set. You can create a set in one step by doing `{3, 4, 5, 6,7, 8, 9, 10, 11, 12}` or `set(range(3, 13))`